guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,29 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
```

Comment:
```suggestion
. On the *Partition Table* tab, associate `{client-provisioning-template-type} default encrypted` to your operating system entry.
```
or
```suggestion
. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` partition table to your operating system entry.
```

Reply:
I'm not sure about "associate ... _to_ ...". AFAIK, the correct preposition is "with". Example: I associate holidays with over-eating. 

Not a blocker.

Reply:
Removed "the", replaced "to" with "with".